app/templates/partials/service_warning.html

```diff
@@ -1,9 +1,9 @@
-{% if true %}
```

Comment:
My guess is it was a previous condition stubbed out and not tidied before commit.